### DISCUSSION TITLE
PatchTool 6.28 11/2

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#241] [Announcements]
更新大部分6.28技能更動，將說話任務改為英文，修正部分翻譯錯誤。

目前漢化至中國服6.15，其餘只有零星漢化。

檔案：
- FFXIVChnTextPatch-20221102.rar：本體。一般而言，下載這個即可。JRE縮小過。

選用：
- rawexd-zhcn-20221102.rar：簡體中文CSV，取代原本resource/rawexd資料夾。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 #90 放到與exe同位置使用。
- backup-20221102.rar：乾淨的0X0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。

目前測試，在之前的更新後，第三方4k字體包可能會導致遊戲崩潰。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v2.70.2'>PatchTool 6.28 11/2</a>.</em>

**Comment by LittleShark718** [comment]
![image](https://user-images.githubusercontent.com/117267657/199450456-88982add-9dd8-4d42-809f-3f4402f861ca.png)
出現這問題怎麼辦 沒中文真的難玩 求求了

**Reply by GpointChen** [reply]
有log嗎？

另外，你也可以嘗試先用backup-20221102.rar裡面的檔案還原後再進行漢化。通常會漢化失敗都是遊戲更新前沒有還原漢化、遊戲檔案損毀所致。

**Comment by Bos-hunting** [comment]
請問今天的更新，可以直接用這個漢化嗎？

**Reply by GpointChen** [reply]
看起來可以 在更新前請記得還原

**Reply by Bos-hunting** [reply]
好的謝謝

**Comment by okami2223** [comment]
作者辛苦了
主線漢化部分有出現亂碼的狀況，但沒有出現閃退錯誤的訊息
![image](https://user-images.githubusercontent.com/44223291/202221707-0f2c1c0f-8ae6-4778-be36-76d35350ba49.png)

**Reply by peckcat** [reply]
> 作者辛苦了 主線漢化部分有出現亂碼的狀況，但沒有出現閃退錯誤的訊息 ![image](https://user-images.githubusercontent.com/44223291/202221707-0f2c1c0f-8ae6-4778-be36-76d35350ba49.png)

這不是亂碼，原本就是這樣。
這是任務本身的設計。

**Reply by okami2223** [reply]
原來如此，抱歉打擾了😅

**Comment by Bos-hunting** [comment]
昨天中國服更新至6.2，希望會有漢化同步至6.2內容，感謝作者辛苦了

**Reply by GpointChen** [reply]
感謝提醒，已更新。
